lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -524,6 +517,51 @@ function dosomething_user_new_user($form, &$form_state) {
 }
 
 /**
+ * Sets the user global attributes and sends them to Northstar.
+ *
+ * @param object $account
+ *   The account to return value for. If NULL, uses global $user.
+ * @param string $country_code
+ *   The country code where the user currently resides. If NULL, uses
+ *   the headers in current request.
+ * @param string $language
+ *   The new language to set on the user. If NULL, gets the matching
+ *   language or defaults.
+ */
+function dosomething_user_set_global_attributes($user = NULL, $country_code = NULL, $language = NULL) {
+  if ($user == NULL) {
+    global $user;
+  }
+  if ($country_code == NULL) {
+    $country_code = dosomething_settings_get_geo_country_code();
+    // If no Fastly headers set, use US EN
+    if ($country_code == NULL) {
+      $language = 'en';
```

Comment:
Yes they are two different languages. en is just for US, en global is for all countries outside of the US we don't explicitly support